tests/data/all_sky_params.yml

```diff
@@ -0,0 +1,32 @@
+# High level match type
+include_perturb_auf: no
+include_phot_like: no
+use_phot_priors: no
+# c/f region definition for photometric likelihood - either "rectangle" for NxM evenly spaced grid points, or "points" to define a list of two-point tuple coordinates, separated by a comma
+cf_region_type: rectangle
+# Frame of the coordinates must be specified -- either "equatorial" or "galactic"
+cf_region_frame: equatorial
+# Maximum separation, in arcseconds, between two sources for them to be deemed positionally correlated
+pos_corr_dist: 11
+# Convolution (fourier transform) parameters
+# Integer number of real space grid points, for Hankel transformations
+real_hankel_points: 10000
+# Integer number of fourier space grid points
+four_hankel_points: 10000
+# Maximum fourier space "rho" parameter considered (typically larger than the inverse of the smallest Gaussian sigma)
+four_max_rho: 100
+# Maximum extent of cross-match, used in non-all-sky cases to remove sources suffering potential edge effects -- min/max first axis coordinates (ra/l) then min/max second axis coordinates (dec/b)
+cross_match_extent:
```

Comment:
`cross_match_extent` is a per-pixel parameter and should be moved to those yml files if/when we include them.